### ISSUE TITLE
CodeInterface builder problem in WIN7 32bit

### ISSUE THREAD

**Issue by sodermax** [picoe/Eto#400]
Hi,

Just noticed that the SetParameters call in CodeInterfaceBuilder.cs does not work in Win7 32 bit environment. Actually the problem is in the 32bit environment. Yes, I still have one computer with WIN 7 32 bit, however that computer is not important to me as I have also 64bit ones. But I still thought that I could write my finding up. 

```
protected virtual void SetParameters(CompilerParameters parameters)
		{
			const string ReferenceAssembliesFolder = @"Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5";

			string referenceDir = null;
			if (EtoEnvironment.Platform.IsWindows)
			{
				referenceDir = Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles(x86)"), ReferenceAssembliesFolder);
				if (!Directory.Exists(referenceDir))
					referenceDir = Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles"), ReferenceAssembliesFolder);
```

The following call will result in ArgumentNullException

```
referenceDir = Path.Combine(Environment.GetEnvironmentVariable("ProgramFiles(x86)"), ReferenceAssembliesFolder);
```

The first parameter results to null and the call to Combine crashes. We should probably check if Environment.........("ProgramFiles(x86)") is null and if thats the case ignore the following combine.path.....("PrograFilesX86") call. 

Best Regards
Max

**Comment by cwensley** [comment]
Hey Max,

Thanks for submitting the issue!  Nice catch there, and thanks for tracking it down!  I'll get that fixed up.

**Comment by cwensley** [comment]
Thanks again!  This should now be fixed in commit 74b2227dfd8db9069e892c0e02c3de3f03479afd
